File: rolemadelen/programmers/120896_한_번만_등장한_문자.cpp
#include <string>
#include <vector>
#include <map>

using namespace std;

string solution(string s) {
    string answer = "";
    map<char, int> mymap;
    for(char ch : s) {
        mymap[ch] += 1;
    }
    
    auto it = mymap.begin();
    while(it != mymap.end()) {
        if(it->second == 1) answer += it->first;
        ++it;
    }
    
    return answer;
}

File: simsu/2주차/120896_한_번만_등장한_문자.js
function solution(s) {
    let answer = [];
    let obj = {};
    for(let str of s) {
        if(!obj[str]) obj[str] = 1;
        else obj[str] ++;
    }
    for (const [key, value] of Object.entries(obj)) {
        if(value==1) answer.push(key);
    }
    return answer.sort().join('');
}